optuna/visualization.py

```diff
@@ -75,11 +75,13 @@ def objective(trial):
         study:
             A :class:`~optuna.study.Study` object whose trials are plotted for their intermediate
             values.
+
+    Returns:
+        A :class:`plotly.graph_objs._figure.Figure` object.
```

Comment:
I think you employed the path to `Figure` used in [this import line](https://github.com/optuna/optuna/pull/704/files#diff-5f61268549ae81f3dd0bd1f4b97a5a74R26), but it would be more suitable to access the public interface.
```suggestion
        A :class:`plotly.graph_objs.Figure` object.
```